assets/alfresco/log4j.properties

```diff
@@ -265,9 +265,9 @@ log4j.logger.org.alfresco.repo.search.impl.noindex.NoIndexIndexer=fatal
 log4j.logger.org.alfresco.repo.search.impl.noindex.NoIndexSearchService=fatal
 log4j.logger.org.alfresco.demoamp.test=DEBUG
 
-log4j.logger.uk.gov.homeoffice.cts.helpers=debug
-log4j.logger.uk.gov.homeoffice.ctsv2.dashboard=debug
+log4j.logger.uk.gov.homeoffice.cts.helpers=info
```

Comment:
would this be better as ERROR

Reply:
This is one of those questions that isn't actually a question isn't it